modules/videopress/shortcode.php

```diff
@@ -1,151 +1,139 @@
 <?php
+
 /**
  * VideoPress Shortcode Handler
  *
  * This file may or may not be included from the Jetpack VideoPress module.
  */
-class Jetpack_VideoPress_Shortcode {
-	public $min_width = 60;
+
+/**
+ * Translate a 'videopress' or 'wpvideo' shortcode and arguments into a video player display.
+ *
+ * Expected input formats:
+ *
+ * [videopress OcobLTqC]
+ * [wpvideo OcobLTqC]
+ *
+ * @link http://codex.wordpress.org/Shortcode_API Shortcode API
+ * @param array $attr shortcode attributes
+ * @return string HTML markup or blank string on fail
+ */
+function videopress_shortcode_callback( $attr, $content = '' ) {
```

Comment:
`$content` is unused in this function, and it's not even mentioned in inline doc so it could be removed.